README.md

```diff
@@ -1,3 +1,13 @@
+To Create DB's 
+CREATE DATABASE chitter;
+CREATE DATABASE chitter_test;
+
+Create tables in DB's
+CREATE TABLE chitter(id SERIAL PRIMARY KEY, peep VARCHAR(60));
+CREATE TABLE chitter_test(id SERIAL PRIMARY KEY, peep VARCHAR(60));
```

Comment:
I think it's confusing to call your databases and tables the same thing. At the moment, you're asking for the user to create two databases, and then for each of those to create two tables within them with the same names (e.g. the database 'chitter_test' will contain the tables 'chitter' and 'chitter_test').

Databases contain tables, tables contain columns.